docs/conf.py

```diff
@@ -34,7 +34,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'JAX'
-copyright = '2019, Google LLC. NumPy and SciPy documentation are copyright the respective authors.'
+copyright = '2020, Google LLC. NumPy and SciPy documentation are copyright the respective authors.'
```

Comment:
Are copyright dates meant to be updated, or meant to reflect the original copyright date? I can never remember, but I think it might be the latter.

Reply:
A quick search shows that lots of people are confused about this. I will revert.